typescript/src/languages/en-US/commands/fun.json

```diff
@@ -209,6 +209,17 @@
 	"wakandaExtended": {
 		"extendedHelp": "Creates an image macro using the [We Don't Do That Here Meme](https://knowyourmeme.com/memes/we-dont-do-that-here) using the given user."
 	},
+	"whereDescription": "I found the person we were looking for!",
+	"whereExtended": {
+		"usages": [
+			"User"
+		],
+		"extendedHelp": "This command generates a meme from [Attack on Titan: The Final Season](https://anilist.co/anime/110277/Shingeki-no-Kyojin-The-Final-Season/), and it's better not to tell anything besides that this will generate a meme of you pointing to a user of your choice.",
```

Comment:
```suggestion
		"extendedHelp": "This command generates a meme from [Attack on Titan: The Final Season](https://anilist.co/anime/110277/Shingeki-no-Kyojin-The-Final-Season/), and it's better not to tell you anything, other than this will generate a meme of you pointing to a user of your choice.",
```